spec/requests/internal/configs_spec.rb

```diff
@@ -97,17 +97,15 @@
       describe "Emails" do
         it "updates email_addresses" do
           expected_email_addresses = {
-            default: "foo@bar.to",
-            business: "partners@dev.to",
-            privacy: "privacy@bar.to",
-            members: "members@bar.to"
+            business: "partners@example.com",
```

Comment:
Since I updated the spec I changed the domain to `example.com` which is a [IANA reserved domain](https://www.iana.org/domains/reserved) unlike `bar.to`.